iphone/Maps/LocalizedStrings/ru.lproj/Localizable.strings

```diff
@@ -1108,138 +834,73 @@
 
 /* To indicate the network of ATMs, bicycle rentals, electric vehicle charging stations... */
 "network" = "Сеть: %@";
-
 "trip_finished" = "Вы прибыли!";
-
 "keyboard_availability_alert" = "Клавиатура недоступна во время движения";
-
 "dialog_routing_change_start_carplay" = "Невозможно построить маршрут от текущего местоположения";
-
 "dialog_routing_change_end_carplay" = "Невозможно построить маршрут до конечной точки. Выберите другую";
-
 "dialog_routing_check_gps_carplay" = "Нет сигнала GPS. Проследуйте на открытую местность";
-
 "dialog_routing_unable_locate_route_carplay" = "Невозможно построить маршрут. Выберите другие точки маршрута";
-
 "dialog_routing_download_files_carplay" = "Чтобы построить маршрут, загрузите недостающие карты на своём устройстве";
-
 "dialog_routing_system_error_carplay" = "Произошла ошибка. Перезапустите приложение";
-
 "dialog_routing_rebuild_from_current_location_carplay" = "Маршрут будет перестроен от вашего текущего местоположения";
-
 "dialog_routing_rebuild_for_vehicle_carplay" = "Маршрут будет изменён на автомобильный";
-
 "not_all_shown_bookmarks_carplay" = "Отображаются не все метки";
-
 "switch_to_phone_bookmarks_carplay" = "Переключитесь на телефон, чтобы увидеть все метки";
-
 "ok" = "Ок";
-
 "speedcams_alert_title_carplay_1" = "Камеры";
-
 "speedcams_alert_title_carplay_2" = "Инфо о камерах";
-
 "download_map_carplay" = "Скачайте карты в приложении Organic Maps на своём устройстве";
-
 "carplay_roundabout_exit" = "%@ съезд";
 
 /* max. 10 symbols, both iOS and Android */
 "sort" = "Сортировать…";
 
 /* iOS */
 "sort_default" = "Сортировать по умолчанию";
-
 "sort_type" = "Сортировать по типу";
-
 "sort_distance" = "Сортировать по расстоянию";
-
 "sort_date" = "Сортировать по дате";
-
 "sort_name" = "Сортировать по имени";
-
 "week_ago_sorttype" = "Неделю назад";
-
 "month_ago_sorttype" = "Месяц назад";
-
 "moremonth_ago_sorttype" = "Больше месяца назад";
-
 "moreyear_ago_sorttype" = "Больше года назад";
-
 "near_me_sorttype" = "Рядом со мной";
-
 "others_sorttype" = "Другие";
-
-
-/********** Bookmark types used for sorting **********/
```

Comment:
Is there a way to have strings somehow grouped in WL?
It'd be great to have some grouping as it provides context to the translators. Alternatively the comment could be duplicated for each string in previous Twine "groups".

Reply:
Both Android and iOS formats are flat, with grouping only possible through comments. It seems that comments like `/********** Bookmark types used for sorting **********/` are actually attached to the following string. Overall, I don’t see this as an issue. Weblate maintains the string order, allowing us to keep these comments in the right place.

Reply:
These comments are produced from Twine's "sections" 
```
[[Bookmark types used for sorting]]

  [food_places]
    tags = android,ios
    en = Food
```
If it would be attached to "food_places" string only then its kinda useless.

Reply:
Maybe we should append all section names to `comment = ` values for all strings inside a section..